src/doc/python_scripting/python.rst

```diff
@@ -324,8 +359,6 @@ For example,
     
         The following documentation is automatically generated from the Python
         source files.  It may be incomplete, incorrect or include features that
-        are considered implementation detail and may vary between Python
-        implementations.  When in doubt, consult the module reference at the
         location listed above.
```

Comment:
Did you mean to cut out this part of the sentence?

Reply:
No...I sure didn't. I recall at first deciding to trim this output section but then unduing that. I guess I didn't undo far enough back. Good catch!